spec/features/providers/allocations/edit_initial_allocation_spec.rb

```diff
@@ -7,6 +7,10 @@
   let(:confirm_deletion_page) { PageObjects::Page::Providers::Allocations::EditInitialAllocations::ConfirmDeletionPage.new }
   let(:allocations_show_page) { PageObjects::Page::Providers::Allocations::ShowPage.new }
 
+  before do
+    Settings.allocations_state = "open"
```

Comment:
by mutating this will you cause issues elsewhere? this may be a better option?
```suggestion
    allow(Settings).to receive(:allocations_state).and_return("open")
```

Reply:
Thats a very good point @asmega. I hadn't thought of that.